### PR TITLE
Adding pull request #6 as well as adding transport and password options

### PR DESCRIPTION
**Pull request by phillpafford**
Fixed a couple of issue as well as adding some new functionality.

I also added this pull request:
- https://github.com/whisller/IrcBotBundle/pull/6

### CODE REVIEW

File: .buildpath [hunk 1]
```diff
@@ -0,0 +1,5 @@
```
Comment (whisller) [inline .buildpath:0]: It shoudn't be in commit.

File: .project [hunk 2]
```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (whisller) [inline .project:1]: It shoudn't be in commit.

File: .settings/org.eclipse.wst.common.project.facet.core.xml [hunk 3]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (whisller) [inline .settings/org.eclipse.wst.common.project.facet.core.xml:1]: It shoudn't be in commit.

**Comment by whisller** [comment]
You have in one commit two features? As I will have a time - in next week - I will check the 6 pull request and merge it.

But if your're adding new functionality it should be done in one commit. One commit == one feature.

And remember about files that are from your IDE and shoudn't be in commit :)

**Comment by whisller** [comment]
Also if you're adding new feature, please add phpunit test for it. Thanks in advice!

**Comment by phillpafford** [comment]
OMG, I didn't realize I checked those in. Still new to git, will redo the push.

Thanks

**Comment by whisller** [comment]
Closing.

**Pull request closed without merging**